src/sections/chat/ChatSection.tsx

```diff
@@ -89,7 +89,7 @@ const ChatSection = ({ chatId, setChatId, loggedIn }) => {
 
             {/* Render unlock profile here if user is not logged in and chat instance is loaded */}
             {userPushSDKInstance && userPushSDKInstance?.readmode() && chatId && (
-              <UnlockProfileWrapper type={UNLOCK_PROFILE_TYPE.BOTTOM_BAR} />
+              <UnlockProfileWrapper type={UNLOCK_PROFILE_TYPE.MODAL} />
```

Comment:
i still see it in the bottom bar, what does this change mean

Reply:
So, when you have chat Id in the url and then you reload when you have not saved PGP keys in storage. You will see a modal instead of Bottom Bar.